plugins/orchestrator-backend/src/service/SonataFlowService.ts

```diff
@@ -193,14 +169,9 @@ export class SonataFlowService {
     definitionId: string;
     serviceUrl: string;
   }): Promise<boolean> {
-    try {
-      const urlToFetch = `${args.serviceUrl}/management/processes/${args.definitionId}`;
-      const response = await fetch(urlToFetch);
-      return response.ok;
-    } catch (error) {
-      this.logger.debug(`Error when pinging workflow service: ${error}`);
-    }
-    return false;
+    const urlToFetch = `${args.serviceUrl}/management/processes/${args.definitionId}`;
```

Comment:
you need to check if response is ok and if not throw an error

Reply:
or is that a separate task?
@gciavarrini

Reply:
this is handled in another story https://issues.redhat.com/browse/FLPATH-1725